### ISSUE TITLE
Extract distinct list of platforms from audit collection in database

### ISSUE THREAD

**Issue by marc0der** [sdkman/sdkman-cli#433]

**Comment by marc0der** [comment]
Extracted the following results:

    466962 - Linux
    284303 - Darwin
     17688 - CYGWIN_NT-6.1
      8198 - posh
      6766 - CYGWIN_NT-6.1-WOW64
      6125 - MINGW32_NT-6.1
      4126 - CYGWIN_NT-6.3
      3457 - SunOS
      2837 - GNU/Linux
      2446 - MINGW32_NT-6.2
      2210 - CYGWIN_NT-6.1-WOW
      2194 - CYGWIN_NT-10.0
      1730 - 
      1145 - MINGW64_NT-6.1
      1065 - MINGW64_NT-10.0
       901 - CYGWIN_NT-5.1
       736 - null
       706 - CYGWIN_NT-6.2
       694 - CYGWIN_NT-10.0-WOW
       523 - CYGWIN_NT-6.3-WOW

In other words, we can switch on:

    "Linux" or "Gnu/Linux"
    "Darwin"
    "CYGWIN_NT*", "MINGW64_NT*"
    "SunOS"

**Comment by marc0der** [comment]
Querying the database for sdkman installs resulted in the following list of `uname -r` results in order of relevance:

* Linux
* Darwin
* CYGWIN_NT-6.1
* CYGWIN_NT-10.0
* MINGW64_NT-6.1
* MINGW32_NT-6.1
* MINGW64_NT-10.0
* CYGWIN_NT-6.1-WOW
* CYGWIN_NT-6.3
* CYGWIN_NT-10.0-WOW
* MINGW32_NT-6.2
* SunOS
* MINGW64_NT-6.3
* CYGWIN_NT-6.3-WOW
* MSYS_NT-6.1
* MSYS_NT-10.0
* CYGWIN_NT-6.1-WOW64
* FreeBSD
* MSYS_NT-6.3
* MINGW32_NT-6.1-WOW
